app/src/main/java/io/btrshop/BtrShopApplication.java

```diff
@@ -24,6 +28,10 @@ public void onCreate() {
                 .applicationModule(new ApplicationModule(this))
                 .apiModule(new ApiModule(mContext.getResources().getString(R.string.base_url_api)))
                 .build();
+
+        // TODO: put your App ID and App Token here
+        // You can get them by adding your app on https://cloud.estimote.com/#/apps
+        EstimoteSDK.initialize(getApplicationContext(), "hamann-denis-gmail-com-s-y-nqc", "2c217585b54fa97b46a39f1b9f3d603f");
```

Comment:
I use this : https://github.com/codepath/android_guides/wiki/Storing-Secret-Keys-in-Android
to put my secret api and token in a secret ressources file, which is in my .gitignore